tests/jest.overrides.js

```diff
@@ -2,11 +2,11 @@ const crypto = require('crypto');
 
 // Cause tests to fail on console.error messages
 // Taken from https://stackoverflow.com/questions/28615293/is-there-a-jest-config-that-will-fail-tests-on-console-warn/50584643#50584643
-let error = console.error;
-console.error = function (message) {
-  error.apply(console, arguments); // keep default behaviour
-  throw (message instanceof Error ? message : new Error(message));
-};
+// let error = console.error;
+// console.error = function (message) {
+//   error.apply(console, arguments); // keep default behaviour
+//   throw (message instanceof Error ? message : new Error(message));
+// };
```

Comment:
Just to double-check, you mean to keep this commented out here, right?

Reply:
I go back and forth on it. But yeah I'm happy enough to leave it commented. It's sometimes useful to uncomment it.